pytest_splunk_addon_ui_smartx/components/controls/single_select.py

```diff
@@ -294,24 +289,23 @@ def get_single_value(self):
                         )
                     }
                 )
-            popoverid = "#" + self.combobox.get_attribute("data-test-popover-id")
+            popover_id = "#" + self.root.get_attribute("data-test-popover-id")
```

Comment:
Couldn't popover_id be defined before if loop once?

Reply:
I would even say it may be defined once per element because it is not changing